### ISSUE TITLE
Links: Better hover state

### ISSUE THREAD

**Issue by EldRoos** [sebgroup/green#977]
### Does the component already exist

- [X] I confirm that I have checked if the component already exists in any of the Green libraries

### Description

Links (including breadcrumbs and section expander) does not have a good hover state, either in Figma or in code.
We (Leopold, Jessi, Jocke, Conny, Viktor) had a meeting 2023-08-23 and decided hover should probably be like this (needs to be tested):

* We decided to keep current (medium blue) color (not changed when hover).
* Normal state is 1px under line and 2px when hovering (growing upwards, animated).
* Links with arrows will still be without underline in normal state and have 2 px underline in hover state.
* Breadcrumbs (missing component) should behave the same as a normal link.
* Section expander should behave the same as a normal link (no background color change like accordion).

### Who will benefit from this feature?

All

**Comment by EldRoos** [comment]
Fixed
